src/cmdc/datasets/official/CA/counties/la.py

```diff
@@ -0,0 +1,84 @@
+import asyncio
+import pandas as pd
+
+from cmdc.datasets.official.base import CountyData
+from cmdc.datasets.puppet import with_page
+
+
+def _class_check(cls):
+    return f"contains(concat(' ',normalize-space(@class),' '),' {cls} ')"
+
+
+async def find_chart_data(page, div_id, colname="y", name_search="number of"):
+    data = None
+    tries = 25
+
+    for i in range(tries):
+        chart_div = await page.waitForSelector(f"#{div_id}")
+        data = await page.evaluate("gd => gd.data", chart_div)
+        layout = await page.evaluate("gd => gd.layout", chart_div)
+        await asyncio.sleep(1)
+        if data is not None:
+            break
+    else:
+        raise ValueError("Did not find data!")
+
+    # find the trace:
+    assert isinstance(data, list)
+    for trace in data:
+        if name_search.lower() in trace["name"].lower():
+            out = pd.DataFrame(
+                {"date": pd.to_datetime(trace["x"]), colname: trace["y"],}
+            )
+
+            return out
+
+    raise ValueError(f"Could not find trace with data {div_id}")
+
+    return dict(data=data, layout=layout)
+
+
+async def test_data():
+    async with with_page() as page:
+        await page.goto(
+            "https://lacdph.shinyapps.io/covid19_surveillance_dashboard/",
+            {"waitUntil": "networkidle2"},
+        )
+
+        # get cases info
+        await page.waitForXPath("//div[@id='myplot']/div")
+        cum_cases = await find_chart_data(page, "myplot", "cases_total", "cases")
+        cum_deaths = await find_chart_data(page, "myplot", "deaths_total", "deaths")
+
+        # move to testing page
+        await (await page.waitForXPath(r"//a[@data-value='testing']")).click()
+
+        # TODO: figure out how to click this...
```

Comment:
Is this TODO important to resolve before merging?

Reply:
oops sorry, changed plan. that should be deleted